storage/src/main/java/com/tencent/rss/storage/util/StorageType.java

```diff
@@ -21,7 +21,8 @@
 public enum StorageType {
   HDFS,
   LOCALFILE,
-  LOCALFILE_AND_HDFS,
+  LOCALFILE_HDFS,
+  LOCALFILE_HDFS_2,
```

Comment:
Is LOCALFILE_HDFS_2 a good name ？
In the previous patch, you use uploaded_HDFS

Reply:
No need.